packages/cli/commands/migrate.ts

```diff
@@ -7,6 +7,7 @@ import { getUserAgent } from '../common.ts';
 export const migrate = new Command('migrate')
 	.description('a CLI for migrating Svelte(Kit) codebases')
 	.argument('<migration>', 'migration to run')
+	.argument('[path]', 'component path or directory to migrate')
```

Comment:
This seems confusing to me because I doubt that all the migrations will accept a path. I think the args should be dependent on the migration. For now I think I'd just pass the args through without trying to list them here

maybe after https://github.com/sveltejs/cli/pull/274 is merged we can use the same CLI parsing library for both and share the args across packages or something

Reply:
That's indeed a valid argument. Now that the migrate code is here, we can think about this directly while implementing #278 

Closing here.